tracer/build/_build/Build.GitHub.cs

```diff
@@ -74,6 +74,40 @@ partial class Build
             Console.WriteLine($"PR assigned");
         });
 
+    Target ClosePreviousMilestone => _ => _
```

Comment:
Wait, have we never been closing them before? 🤔  That seems strange 🤷‍♂️

Reply:
There was a step in the release process to do it manually.